projects/plugins/jetpack/extensions/blocks/videopress/videopress.php

```diff
@@ -41,19 +41,3 @@ function load_assets( $attrs, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( FEATURE_FOLDER );
 	return $content;
 }
-
-// Set the videopress/video feature availability.
-add_action(
-	'jetpack_register_gutenberg_extensions',
-	function () {
-		\Jetpack_Gutenberg::set_extension_available( 'videopress/video' );
-	}
-);
-
-// Set the videopress/video-chapters feature availability.
-add_action(
```

Comment:
By removing this we are also unregistering the Video Chapters block, right?

Reply:
yes, the Video Chapters block has been removed completely in fact